AllReadyApp/Web-App/AllReady/Caching/OptimizeRouteResultFromCacheQueryHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿using AllReady.Caching;
+using AllReady.Services.Mapping.Routing;
+using MediatR;
+using Microsoft.Extensions.Caching.Memory;
+
+namespace AllReady.Areas.Admin.Features.Itineraries
+{
+    public class OptimizeRouteResultFromCacheQueryHandler : IRequestHandler<OptimizeRouteResultFromCacheQuery, OptimizeRouteResultStatus>
```

Comment:
I went back on forward on this. I wasn't sure if I was misusing a handler here but I think in the end it's okay. Name wise, I'm happy to change it. I wasn't sure if we wanted to distinguish what to actually expect here. Perhaps... CachedOptimzeRouteResultQuery?